### PR TITLE
 #164797142 Configured api documentation with swagger

### PR DESCRIPTION
**Pull request by emmsdan**
#### What does this PR do?
 - auto-generates documentation from route comments
 - custom theme for swagger documentation

#### Description of Task to be completed?
 - here is the **wiki** on how to use swagger-jsdoc comment in your api route [https://github.com/andela/zinnia-ah-backend/wiki/Using-Swagger-API-Documentation-in-your-Project](https://github.com/andela/zinnia-ah-backend/wiki/Using-Swagger-API-Documentation-in-your-Project)

#### How should this be manually tested?

* git clone `https://github.com/andela/zinnia-ah-backend.git`
* run `git checkout ft-api-documentation-164797142`
* pull from the corresponding remote branch
* run `npm install`
* comment on a route using [this wiki as a guide](https://github.com/andela/zinnia-ah-backend/wiki/Using-Swagger-API-Documentation-in-your-Project).
* run `npm run start:dev`
* open `http://localhost:3000/documentation`

#### What are the relevant pivotal tracker stories?(if applicable)
#164797142

#### Screenshots
Screenshot 2019-03-28 at 11 19 57 AM

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (Japhethca) [inline .vscode/launch.json]: It seems like this file was added in your recent commits, I don't think this is supposed to be committed. Is there any reason why you pushed this?

File: .env.example [hunk 2]
```diff
@@ -1 +1,2 @@
 DATABASE_DEV_URL = postgres://username:password@localhost:5432/database_name
+HOST_URL =
```
Comment (Japhethca) [inline .env.example]: There should be a new line here.

File: .vscode/launch.json (hunk 1 above)
Comment (emmsdan) [reply .vscode/launch.json]: I am currently working on it now. thanks

File: .env.example (hunk 2 above)
Comment (emmsdan) [reply .env.example]: noted.

**Review by Japhethca — DISMISSED** [review]
Nice!!

**Review by Japhethca — APPROVED** [review]
Looks good!

**Pull request merged**